Plugins/IngameDebugConsole/IngameDebugConsole.Runtime.asmdef

```diff
@@ -1,3 +1,6 @@
 ﻿{
-	"name": "IngameDebugConsole.Runtime"
+	"name": "IngameDebugConsole.Runtime",
+	"references": [
+		"Unity.InputSystem"
+	]
```

Comment:
But won't this throw error messages if the new input system isn't available or enabled on a Unity project?

Reply:
Without this, the package does not have access to the new Input System classes and doesn't compile at all. Package.json add a dependency to the new Input System BUT don't need to activate it.

Reply:
We must test if this breaks compatibility with Unity versions 2017.3 to 2018.2 where Assembly Definition Files exist but apparently, new Input System doesn't.